CommunityToolkit.WinUI.UI/Converters/ResourceNameToResourceStringConverter.cs

```diff
@@ -18,7 +18,7 @@ namespace CommunityToolkit.WinUI.UI.Converters
     /// </summary>
     public sealed class ResourceNameToResourceStringConverter : IValueConverter
     {
-        private readonly ResourceLoader _resourceLoader = new ResourceLoader();
```

Comment:
Looks like we should be fixing uno instead (there's an issue opened IIRC). If I recall properly, `ResourceLoader.GetForViewIndependentUse();` is not available under WinAppSDK.

Reply:
cc: @jeromelaban

Reply:
The WinUI branch has this change: https://github.com/CommunityToolkit/WindowsCommunityToolkit/blob/080bdb99e6810eb2157ad635f00e64a64acabb50/CommunityToolkit.WinUI.UI/Converters/ResourceNameToResourceStringConverter.cs#L16. We'll need to look at the default instance for the resource loader then.